packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -202,58 +202,67 @@
 
 :host([selection-appearance="border"][layout="horizontal"]) {
   .container.selected {
-    box-shadow: inset 0 -4px 0 0 var(--calcite-color-brand);
+    box-shadow: inset 0 -4px 0 0 var(--calcite-tile-accent-color-press, var(--calcite-color-brand));
   }
 }
 
 :host([selection-appearance="border"][layout="vertical"]) {
   .container.selected {
-    box-shadow: inset 4px 0 0 0 var(--calcite-color-brand);
+    box-shadow: inset 4px 0 0 0 var(--calcite-tile-accent-color-press, var(--calcite-color-brand));
+  }
+}
+
+:host(:hover:not([disabled])),
+:host([active]:not([disabled])) {
+  .heading {
+    color: var(--calcite-tile-heading-text-color, var(--calcite-color-text-1));
+  }
+  .description {
+    color: var(--calcite-tile-text-color, var(--calcite-color-text-2));
   }
 }
 
 :host([href]:focus:not([disabled])),
 :host([href]:hover:not([disabled])) {
-  --calcite-tile-border-color: var(--calcite-color-text-link);
-  --calcite-tile-heading-text-color: var(--calcite-color-text-link);
-  --calcite-icon-color: var(--calcite-color-text-link);
   .container {
```

Comment:
the "href" tile is an edge case I think. The design pattern changes significantly when a tile becomes a "link" tile however it can still exist alongside non-link tiles. To define this I think we need to add a new token for this specific case. `--calcite-tile-link-color`

```css
:host([href]:focus:not([disabled])),
:host([href]:hover:not([disabled])) {
  .container {
    outline-color: var(--calcite-tile-link-color, var(--calcite-color-text-link));
    position: relative;
    z-index: var(--calcite-z-index);
  }
  .icon {
    color: var(--calcite-tile-link-color, var(--calcite-color-text-link));
  }
  .heading {
    color: var(--calcite-tile-link-color, var(--calcite-color-text-link));
  }
}

:host([href]:active:not([disabled])) {
  .container {
    box-shadow: 0 0 0 3px var(--calcite-tile-border-color);
    box-shadow: inset 0 0 0 1px var(--calcite-tile-link-color, var(--calcite-color-text-link));
    outline-color: var(var(--calcite-tile-link-color, var(--calcite-color-text-link));
  }
}
```

Reply:
done :)